src/arena/SprintCreation.js

```diff
@@ -166,9 +166,9 @@ function SprintCreation(props) {
           cloneDeep(finalDBMission),
         ],
       };
+      debugger;
```

Comment:
Can you please remove this? Very cool though, I actually didn't know that functionality was built into VS Code... thanks for teaching me something new!